src/System.Security.Cryptography.Pkcs/pkg/System.Security.Cryptography.Pkcs.pkgproj

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Security.Cryptography.Pkcs.csproj">
+      <SupportedFramework>net46;net461;netcore50;netstandardapp1.5</SupportedFramework>
```

Comment:
(updated PR branch)

So if understand, "netstandard 1.x" replaces "<empty-TargetGroup>+<netcore>" and we can use this as long as the same binary works for both (i.e. no "#if"'s or conditional includes of sources.)

Well right now, the assembly is in that happy state (due to my finding a way around the lack of NCryptIsKeyHandle() on UWP), but I'm not sure we'll continue to be so lucky in the future if we start adding stuff to it. So my question is, what's the downside to keeping it the way it is?